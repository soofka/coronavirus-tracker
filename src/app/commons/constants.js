const VIRUS_DATA_API_BASE_URL = 'https://covid-tracker-us.herokuapp.com/v2/';

export const GLOBAL_VIRUS_DATA_API_URL = `${VIRUS_DATA_API_BASE_URL}latest`;
export const LATEST_REGIONAL_VIRUS_DATA_API_URL = `${VIRUS_DATA_API_BASE_URL}locations`;
export const HISTORICAL_REGIONAL_VIRUS_DATA_API_BASE_URL = `${VIRUS_DATA_API_BASE_URL}locations/`;
export const ASSETS_BASE_URL = 'assets/';

export const LOCALES = {
  en_EN: 'en_EN',
  pl_PL: 'pl_PL',
};
export const DEFAULT_LOCALE = LOCALES.en_EN;

export const DATA_GRANULARITIES = {
  DAILY: 'daily',
  WEEKLY: 'weekly',
  MONTHLY: 'monthly',
};

export const DEFAULT_DATA_GRANULARITY = DATA_GRANULARITIES.DAILY;
export const DEFAULT_REGION = 'none';
export const DEFAULT_DATE = 0;