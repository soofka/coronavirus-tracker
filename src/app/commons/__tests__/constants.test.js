import * as constants from '../constants';

describe('constants.js', () => {
  it('did not change', () => {
    expect(constants).toMatchSnapshot();
  });
});
