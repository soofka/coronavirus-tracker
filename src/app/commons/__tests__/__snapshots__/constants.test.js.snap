// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants.js did not change 1`] = `
Object {
  "ASSETS_BASE_URL": "assets/",
  "DATA_GRANULARITIES": Object {
    "DAILY": "daily",
    "MONTHLY": "monthly",
    "WEEKLY": "weekly",
  },
  "DEFAULT_DATA_GRANULARITY": "daily",
  "DEFAULT_DATE": 0,
  "DEFAULT_LOCALE": "en_EN",
  "DEFAULT_REGION": "none",
  "GLOBAL_VIRUS_DATA_API_URL": "https://covid-tracker-us.herokuapp.com/v2/latest",
  "HISTORICAL_REGIONAL_VIRUS_DATA_API_BASE_URL": "https://covid-tracker-us.herokuapp.com/v2/locations/",
  "LATEST_REGIONAL_VIRUS_DATA_API_URL": "https://covid-tracker-us.herokuapp.com/v2/locations",
  "LOCALES": Object {
    "en_EN": "en_EN",
    "pl_PL": "pl_PL",
  },
}
`;
